packages/app/src/App.tsx

```diff
@@ -109,8 +109,14 @@ import { TwoColumnLayout } from './components/scaffolder/customScaffolderLayouts
 import { ScoreBoardPage } from '@oriflame/backstage-plugin-score-card';
 import { StackstormPage } from '@backstage/plugin-stackstorm';
 import { PuppetDbPage } from '@backstage/plugin-puppetdb';
+import { locales } from './i18nLocales';
 
 const app = createApp({
+  localeConfig: {
+    lng: 'zh',
```

Comment:
Yes, I was use the i18next config type to `localConfig`.  defaultLanguage seems to be `fallbackLng` in i18next. Because of i18n function base on i18next, so i hope config has no gap with i18next. If someone know i18next, he also know backstage i18n